tests/test_encoding.py

```diff
@@ -309,17 +310,17 @@ def test_issue_70(self):
             "bit_mask": None,
             "is_condensed_raw": None,
         }
-        uint2 = StandardLengthType(bit_length=2, **unit_kwargs)
-        uint1 = StandardLengthType(bit_length=1, **unit_kwargs)
-        param_kwargs = {
+        uint2 = StandardLengthType(bit_length=2, **unit_kwargs)  # type: ignore[arg-type]
+        uint1 = StandardLengthType(bit_length=1, **unit_kwargs)  # type: ignore[arg-type]
```

Comment:
Why?

Reply:
mypy gets confused by the keyword argument dictionary...